src/lib.rs

```diff
@@ -196,7 +196,7 @@ pub enum InterpolationType {
 pub trait Resampler<T> {
     /// Resample a chunk of audio. Input and output data is stored in a vector,
     /// where each element contains a vector with all samples for a single channel.
```

Comment:
It would be nice if this help text mentioned that the inner vectors are AsRef<[T]>. 

Something like this?
```
Resample a chunk of audio. The input data is should be a slice, where each element is a reference to a slice (`AsRef<[T]>`) that holds the samples for a single channel. Since Vec<T> implements AsRef<[T]>, the input can be simply &Vec<Vec<T>>.

The output data is a vector, where each element contains a vector holding the samples for one channel.  
```

Reply:
How's my attempt in 29a5d1b?

Reply:
Looks great! Merging :)